crypto/src/main/scala/coop/rchain/crypto/signatures/Ed25519.scala

```diff
@@ -10,11 +11,11 @@ object Ed25519 extends SignaturesAlg {
   val name: String = "Ed25519".toLowerCase
 
   //TODO: Make use of strongly typed keys
-  def newKeyPair: (Array[Byte], Array[Byte]) = {
+  def newKeyPair: (PrivateKey, PublicKey) = {
```

Comment:
Shouldn't other methods of SignaturesAlg take and return the PrivateKey/PublicKey wrappers as well?

Reply:
I guess I'm fine with sticking to what's done in this PR and scheduling this work for later though.

Reply:
They do, actually. They're implemented in `SignaturesAlg`, since they're just wrappers.